documentation/commands/mutation.md

```diff
@@ -33,6 +33,10 @@ The metadata is located after the items value. The size for the value is therefo
 
 NRU is an internal field used by the server and may safely be ignored by other consumers.
 
+If the datatype contains the flag to indicate that the document carries
+xattrs the first 4 bytes in the document specified the length of the
```

Comment:
There was some discussion of encoding this in the extras (at least for GET). Should we do a similar thing for DCP_MUTATION?